indexer/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
+
+## [Unreleased]
+
+### Features
+
+* [#473](https://github.com/dydxprotocol/v4-chain/pull/473/files) Changelog example entry.
```

Comment:
Can we add an actual example per section? For example, this one https://github.com/dydxprotocol/v4-chain/pull/481 for state breaking.

Also, this PR doesn't feel like a good example. Can you pick a different PR for the features section?